src/components/create-user.html

```diff
@@ -1,95 +1,136 @@
+        clear: both;
+        display: table;
+      }
+
+      .card {
+        box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+        padding-top: initial;
+      }
+    </style>
+
+    <div class="card">
+      <div class="container">
         <h1 class="div-center">Create User</h1>
+        <form>
+          <label for="fname">First Name</label>
+          <input type="text" placeholder="Your name.." value="{{user.firstName::input}}">
+
+          <label for="lname">Last Name</label>
+          <input type="text" placeholder="Your last name.." value="{{user.lastName::input}}">
+
+          <label for="phone">Phone</label>
+          <input type="text" placeholder="Your phone number.." value="{{user.phone::input}}">
+
+          <label for="email">Email</label>
+          <input type="text" placeholder="Your email.." value="{{user.email::input}}">
+        </form>
         <div>
-            <div class="container">
-                <div class="form-border">
-                    <form>
-                        <label for="fname">First Name</label>
-                        <input type="text" id="fname" name="firstname" placeholder="Your name..">
-
-                        <label for="lname">Last Name</label>
-                        <input type="text" id="lname" name="lastname" placeholder="Your last name..">
-
-                        <label for="phone">Phone</label>
-                        <input type="text" id="phone" name="phone" placeholder="Your phone number..">
-
-                        <label for="email">Email</label>
-                        <input type="text" id="email" name="email" placeholder="Your email..">
-                        <div>
-                            <button class="button">Create</button>
-                        </div>
-                    </form>
-                </div>
-            </div>
+          <button class="button" on-click="createUser">Create User</button>
         </div>
-    </template>
+      </div>
+    </div>
+  </template>
 
-    <script type="module">
+  <script type="module">
+    import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+    import { LocalStorage } from '../api/localStorage.js';
 
-        import { Element as PolymerElement } from '@banno/polymer/polymer-element.js';
+    const Storage = new LocalStorage();
 
-        class CreateUser extends PolymerElement {
-            
-            static get is() { return 'create-user'; }
+    class CreateUser extends PolymerElement {
+      static get is() { return 'create-user'; }
+
+      static get properties() {
+        return {
+          user: {
+            type: Object,
+            value() {
+              return {
+                firstName: '',
+                lastName: '',
+                phone: '',
+                email: '',
+                id: ''
+              }
+            }
+          }
         }
+      }
 
-        customElements.define(CreateUser.is, CreateUser);
+      getUserIndex() {
```

Comment:
again, it doesn't look like you're using this function at all here. let's get rid of it.